docs/quick_reference/consensus_help.txt

```diff
@@ -1,9 +1,9 @@
-                                                                                               
- Usage: taxpasta consensus [OPTIONS]                                                           
-                                                                                               
- Form a consensus for the same sample but from different taxonomic profiles.                   
-                                                                                               
-╭─ Options ───────────────────────────────────────────────────────────────────────────────────╮
-│ --help  -h        Show this message and exit.                                               │
-╰─────────────────────────────────────────────────────────────────────────────────────────────╯
+                                                                                              
+ Usage: taxpasta consensus [OPTIONS]                                                          
+                                                                                              
+ Form a consensus for the same sample but from different taxonomic profiles.                  
```

Comment:
Should this message say 'not implemented' or something?

Reply:
Hmm, probably 😛 The actual documentation page does so but the command should, too, or be removed for now.